src/pages/patterns/lead-space.mdx

```diff
@@ -40,7 +40,7 @@ When designed and implemented effectively, the Lead space can be used to engage
 
 Here are some of the elements that are commonly used and currently available. As we continue to build and offer additional options, we will share new designs on this page.
 
-![Anatomy of the lead space](./images/lead-space/LeadSpace_Left_Heading06_Anatomy.jpg)
+![Anatomy of the lead space](../../images/pattern/leadspace/leadspace_patterns_anatomy_dark_demo.jpg)
```

Comment:
Seeing as this is the pattern and not the component should we explore adding tags, breadcrumbs, and maybe search?

Reply:
Agree. In the next update, we will have to.